docs/schemas/FeatureManagement.v3.0.0.json

```diff
@@ -0,0 +1,150 @@
+    "$id":"http://azconfig.io/schemas/FeatureManagement-DotNet/v3.0.0/FeatureManagement.json", 
+    "title":"Decalaration of features in the Microsoft.FeatureManagement library.",
+    "definitions": { 
+        "FeatureFlag": { 
+            "type":"object", 
+            "properties": { 
+                "EnabledFor": { 
+                    "type":"array", 
+                    "items": { 
+                        "type":"object", 
+                        "required":[ 
+                            "Name" 
+                        ], 
+                        "properties": { 
+                            "Name": { 
+                                "type":"string" 
+                            }, 
+                            "Parameters": { 
+                                "type":"object", 
+                                "patternProperties": { 
+                                    "^.*$": { 
+                                        "anyOf":[ 
+                                            { 
+                                                "type":"string" 
+                                            }, 
+                                            { 
+                                                "type":"null" 
+                                            }, 
+                                            { 
+                                                "type":"object" 
+                                            }, 
+                                            { 
+                                                "type":"number" 
+                                            }, 
+                                            { 
+                                                "type":"array" 
+                                            }, 
+                                            { 
+                                                "type":"boolean" 
+                                            } 
+                                        ] 
+                                    } 
+                                }, 
+                                "additionalProperties":false 
+                            } 
+                        } 
+                    } 
+                } 
+            } 
+        }, 
+        "DynamicFeature": { 
+            "type":"object", 
+            "required": [ 
+              "Assigner", 
+              "Variants" 
+            ], 
+            "properties": { 
+                "Assigner": { 
+                    "type":"string" 
+                }, 
+                "Variants": { 
+                    "type":"array", 
+                    "items": { 
+                        "type":"object", 
+                        "required":[ 
+                            "Name", 
+                            "ConfigurationReference" 
+                        ], 
+                        "properties": { 
+                            "Default": { 
+                                "type":"boolean" 
```

Comment:
I remember we require one of the variants to be default. Not sure how to enforce it.

Reply:
I did a search, it doesn't look like a constraint that can be enforced via JSON schema.